docs/docs/guides/labs/components/integrations/airflow-component-tutorial.md

```diff
@@ -2,19 +2,24 @@
 title: Dagster & Airflow with components
 sidebar_position: 300
 description: The dagster-airlift library provides an AirflowInstanceComponent, which you can use to peer a Dagster project with an Airflow instance.
+unlisted: true
 ---
 
 import DgComponentsPreview from '@site/docs/partials/\_DgComponentsPreview.md';
 
 <DgComponentsPreview />
 
-The [dagster-airlift](/integrations/libraries/airlift) library provides an `AirflowInstanceComponent` which can be used to represent Airflow DAGs in Dagster.
+The [dagster-airlift](/integrations/libraries/airlift) library provides an `AirflowInstanceComponent` which can be used to represent Airflow DAGs in Dagster, allowing easy interoperability between Airflow and Dagster.
 
 ## Setup and peering
 
 ### 1. Create a components-ready Dagster project
 
-To begin, you'll need a components-ready Dagster project. To create one, follow the steps in "[Creating a project with dg](https://docs.dagster.io/guides/labs/dg/creating-a-project)".
+To begin, you'll need a components-ready Dagster project. To create one, use the `uvx create-dagster` command:
+
+```
+uvx create-dagster project my-project && cd my-project/src
```

Comment:
@benpankow the Fivetran and dlt component guides use `uvx`, but the Airbyte one doesn't -- not sure what we want to standardize on here.